nfc/libs/NfcCoreLib/lib/Nci/phNciNfc_Init.c

```diff
@@ -42,6 +51,9 @@ phNciNfc_SequenceP_t gphNciNfc_NfccResetSequence[] = {
     {NULL, &phNciNfc_CompleteNfccResetSequence}
 };
 
+/** Core Reset notification timeout */
+#define PHNCINFC_CORE_RESET_NTF_TIMEOUT        (30)
+
```

Comment:
Units should probably be indicated in macro name (is this ms?)

Also, some reasoning should be provided as to why this is enough time to allow for reset (eg. is this a spec dictated number?)